api-stress/report/build_report.py

```
@@ -0,0 +1,336 @@
+    json.dump(findings, open(run_dir / "findings.json", "w"), indent=2)
+
+    # ----- counts -----
+    p = lambda x: sum(1 for f in findings if f["priority"] == x)  # noqa: E731
+    tested = len(summary)
+    total_5xx_eps = sum(1 for r in summary.values() if r["n5xx"])
+    slowest = sorted(summary.items(), key=lambda kv: -kv[1]["p95"])[:15]
+    authz = [f for f in findings if f["kind"] == "authz"]
+    idor = [f for f in findings if f["kind"] == "idor"]
+
+    md = []
+    md.append(f"# PROMPT API Stress & Fuzz Report")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)